tests/utils/helpers.py

```diff
@@ -17,6 +20,45 @@
 )
 
 
+def rlp_encode_tx(tx: dict) -> bytes:
+    if "type" in tx:
+        typed_transaction = TypedTransaction.from_dict(tx)
+
+        sanitized_transaction = transaction_rpc_to_rlp_structure(
+            typed_transaction.transaction.dictionary
+        )
+
+        # RPC-structured transaction to rlp-structured transaction
+        rlp_serializer = (
+            typed_transaction.transaction.__class__._unsigned_transaction_serializer
+        )
+        encoded_unsigned_tx = pipe(
+            rlp_serializer.from_dict(sanitized_transaction),  # type: ignore  # noqa: E501
+            lambda val: rlp.encode(val),  # rlp([...])
```

Comment:
you don't need to create a `lambda` here, but using just `rlp.encode` should work